.github/workflows/rails_ci.yml

```diff
@@ -5,37 +5,43 @@
 # This workflow will install a prebuilt Ruby version, install dependencies, and
 # run tests and linters.
 name: "Ruby on Rails CI"
-on:
-  push:
-    branches: [ $default-branch, 2024-upgrades-main ]
-  pull_request:
-    branches: [ $default-branch, 2024-upgrades-main ]
+on: push
+  # push:
+  #   branches: [ $default-branch, 2024-upgrades-main ]
+  # pull_request:
+  #   branches: [ $default-branch, 2024-upgrades-main ]
 jobs:
   test:
     runs-on: ubuntu-latest
     env:
       RAILS_ENV: test
-      DATABASE_URL: "postgres://rails:password@localhost:5432/rails_test"
-      aws_access_key_id: test
-      aws_secret_access_key: test
+
+      db_host: localhost
```

Comment:
I'd like to convert these to be more canonical rails (eg DATABASE_PASSWORD) but for now I want to minimize changes and it's what we already have in secrets

Reply:
yeah good idea, will do!